tests_v2/test_ssh_remote_docker_forwarding.py

```
@@ -0,0 +1,651 @@
+            _must(bool(workspace_id_shared), f"cmux ssh output missing workspace_id for shared transport test: {payload_shared}")
+
+            _, shared_proxy_port = _wait_connected_proxy_port(client, workspace_id_shared)
+            _must(
+                shared_proxy_port == proxy_port,
+                f"identical SSH transports should share one local proxy endpoint: {proxy_port} vs {shared_proxy_port}",
+            )
+
+            try:
+                client.close_workspace(workspace_id_shared)
+            except Exception:
+                pass
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)